controller/pagecontroller.php

```diff
@@ -96,6 +92,35 @@ public function index() {
 	}
 
 	/**
+	 * Returns the parameters to be used in the index function
+	 *
+	 * @param $appName
+	 *
+	 * @return array<string, string>
+	 */
+	private function getParameters($appName) {
```

Comment:
I think we should be more specific, like `getIndexParameters`. The reason being that almost all methods on this page have parameters, so if we need to do this another time this would have to be changed.